### DISCUSSION TITLE
Add some additional thread name tests

### DISCUSSION THREAD

**Discussion by johnsonjh** [aremmell/libsir#477] [Ideas]
* Add some extra thread set/get name tests, like setting a NULL name, setting a very very long name, etc.

**Comment by StoneCypher** [comment]
1. the sweet baby jesus wants these to be [stochastic tests](https://www.youtube.com/watch?v=hXnS_Xjwk2Y)
2. why does github think i want eleventy four emails about rimmel converting these into discussions this morning
